README.md

```diff
@@ -295,19 +313,9 @@ background = Image.open(output[0])
 
 ### FileOutput
 
-Is a file-like object returned from the `replicate.run()` method that makes it easier to work with models
-that output files. It implements `Iterator` and `AsyncIterator` for reading the file data in chunks as well
-as `read` and `aread()` to read the entire file into memory.
-
-Lastly, the underlying datasource is available on the `url` attribute.
-
-> [!NOTE]
-> The `url` attribute can vary between a remote URL and a data-uri depending on whether the server has
-> optimized the request. For small files <5mb using the syncronous API data-uris will be returned to
-> remove the need to make subsequent requests for the file data. To disable this pass `wait=false`
-> to the replicate.run() function.
+Is a file-like object returned from the `replicate.run()` method that makes it easier to work with models that output files. It implements `Iterator` and `AsyncIterator` for reading the file data in chunks as well as `read` and `aread()` to read the entire file into memory.
```

Comment:
Can we link `file-like object` to https://docs.python.org/3/glossary.html#term-file-object?